packages/library/index.js

```diff
@@ -56,7 +56,7 @@ module.exports = (neutrino, opts = {}) => {
   neutrino.config
     .when(hasSourceMap, () => neutrino.use(banner))
     .devtool('source-map')
-    .externals([nodeExternals()])
+    .externals([nodeExternals(options.externals)])
```

Comment:
`options.externals` isn't defined by default, so this will be calling `nodeExternals(undefined)`. This is currently handled ok, due to:
https://github.com/liady/webpack-node-externals/blob/cfa1c5b33752fb8cb72360167ae89b23816cdefe/index.js#L22

...however it doesn't feel right to be relying on an implementation detail that they could possibly change in a minor version.

As such, perhaps the implementation should copy what `react-components` does? (With optional cleanup there, since the current `opts.externals !== false && {}` overly complex, unless I'm mis-remembering how deepmerge handles merging different data types.)

Reply:
Thank you for updating the PR - it's missing the `.when()` of the react-components implementation though?
https://github.com/mozilla-neutrino/neutrino-dev/blob/ed21fa6074354609389710f051286356b073cb4e/packages/react-components/index.js#L53